index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    <title>Awesome Books</title>
+</head>
+<body>
+    <main>
+        <h1>Awesome Books</h1>
+        <ul id="books-list"></ul>
+        <form id="book-form">
+            <input type="text" id="title" placeholder="Title"><br>
+            <input type="text" id="author" placeholder="Author"><br>
+            <button id="submit">Add</button>
+        </form>
```

Comment:
- [ ] [OPTIONAL] Everything is fine about the projects 👏 , however I noticed that you when you are adding a book while the inputs are empty, your app is still adding a book, kindly make sure you add some input validations before adding 👍